pyccel/ast/headers.py

```diff
@@ -654,23 +656,9 @@ def apply(self, args, results=None):
                     raise ValueError('variable not allowed after an optional argument')
 
             for arg,val in zip(self.arguments[:len(sorted_args)],sorted_args):
```

Comment:
I don't think so. As I understand it. This section collects all the `*args` (sorted_args) from the `args` input, and the next loop collects all the `**kwargs` (unsorted_args) from the `args` input. That said this seems to be a useless step as it could be done directly above

Reply:
Fixed 60c2b5b9